includes/profile.php

```diff
@@ -355,6 +362,22 @@ function pmpro_membership_level_profile_fields_update()
 			$pmproemail->sendAdminChangeEmail(get_userdata($user_ID));
 		}
 	}
+
+	//Refund their most recent subscription
+	if( !empty( $_REQUEST['refund_subscription'] ) ) {
```

Comment:
I don't see anywhere where we are checking a nonce on this handling or any of the above `$_REQUEST` handling

Reply:
Im working off of existing code.. should we not create an issue for that to be handled later?